projects/extension/sql/idempotent/012-vectorizer-int.sql

```diff
@@ -619,9 +730,9 @@ set search_path to pg_catalog, pg_temp
 do $upgrade_block$
 declare
     _vec record;
-    _new_version text := '0.8.1';
+    _new_version text := '0.10';
```

Comment:
This should just recreate all triggers with the new functions but I should probably test this.